internal/tui/list.go

```diff
@@ -241,6 +247,9 @@ func (m *ListModel) Update(msg tea.Msg) (Component, tea.Cmd) {
 			return editModel, nil
 		case "s":
 			return m, func() tea.Msg { return scanMsg{} }
+		case "S":
+			_, cmd := m.scanner.Toggle()
```

Comment:
We want to render a status message that says "Active scan disabled" or enabled respectively. Let's make a special statusMsg struct that contains the message string we need to set that we can return. Use this to set the status message.